packages/Uploader/src/helpers.js

```diff
@@ -115,8 +115,25 @@ export function getFiles({ event, maxFileSize, okFileTypes, endpoint }) {
       files = event.target.files;
     }
 
-    return createFilesDataStructure({ files, maxFileSize, okFileTypes, endpoint });
+    return createFilesDataStructure({ files, maxFileSize, supportedMimeTypes, endpoint });
   }
 
   return [];
 }
+
+export function getNumberWithUnits(number) {
+  if (number > 1024) {
+    if (number > 1024 * 1024) {
+      if (number > 1024 * 1024 * 1024) {
+        if (number > 1024 * 1024 * 1024 * 1024) {
+          return `${(number / (1024 * 1024 * 1024 * 1024)).toFixed(3)}TiB`;
```

Comment:
`TiB` should be localized, no?

Reply:
I don't think you can translate `Mebibyte` can you?  Can you translate that acronym? mebibyte would be translate to `mega binary byte` so would make sense to translate TiB to `tera binary byte`? which would make really stranger to read. But to be honest I haven't seen this word translated it.

Reply:
are you kidding me french?  `Megabyte` => `mégaoctet`